server/game/cards/characters/05/timettsonofsimett.js

```diff
@@ -0,0 +1,38 @@
+const DrawCard = require('../../../drawcard.js');
+
+class TimettSonOfTimett extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: (event, challenge) => (
+                    challenge.winner === this.controller && 
+                    challenge.isAttacking(this))
+            },
+            handler: () => {
+                this.game.promptForSelect(this.controller, {
+                    cardCondition: card => (
+                        card.location === 'play area' && 
+                        card.getType() === 'character' && 
+                        card.getCost() <= this.numberOfClansmen()),
+                    activePromptTitle: 'Select character to kill',
+                    waitingPromptTitle: 'Waiting for opponent to use ' + this.name,
+                    onSelect: (player, card) => (
+                        card.controller.killCard(card),
+                        this.game.addMessage('{0} uses {1} to kill {2}', this.controller, this, card))
+                });
+            }
+        });
+    }
+
+    numberOfClansmen() {
```

Comment:
minor nitpick since you need to change the filename anyway, please rename this to getNumberOfClansmen